test/parallel/test-runner-run.mjs

```diff
@@ -465,6 +469,19 @@ describe('require(\'node:test\').run', { concurrency: true }, () => {
           code: 'ERR_INVALID_ARG_TYPE'
         }));
     });
+
+    it('should pass instance of stream to setup', async () => {
+      const stream = run({
+        files: [join(testFixtures, 'default-behavior/test/random.cjs')],
+        setup: common.mustCall((root) => {
+          assert(root instanceof TestsStream);
```

Comment:
```suggestion
          assert.isEqual(root.constructor.name, "TestsStream");
```

Reply:
`.constructor` is a lot less reliable or idiomatic than `instanceof` (which is not reliable either); what's this attempting to verify?

Reply:
Just trying to avoid `--expose-internal`, but not very important